image_data_guide/01_download_data.ipynb

```diff
@@ -0,0 +1,546 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Downloading and Combining Datasets (Part 1/4)\n",
+    "**Download** | Structure | Preprocess | Train"
+   ]
+  },
+  {
```

Comment:
Provide a high-llevel overview of what the notebook does. e.g. "In this notebook you will explore how to download, a dataset from the [blank]. To mimic the real-world, you will also see how to download new data and combine that with your dataset..."

Reply:
Added a brief into at the top of the notebook